src/app/my-info/(account)/cart/_components/CustomCardList.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import { useQuery } from '@tanstack/react-query';
+import classNames from 'classnames/bind';
+
+import { getCartData } from '@/api/cartAPI';
+import type { CartAPIDataType } from '@/types/CartTypes';
+import CartCard from './CartCard';
+
+import styles from './CustomCardList.module.scss';
+
+const cn = classNames.bind(styles);
+
+export default function CustomCardList() {
+  const { data, isSuccess } = useQuery({ queryKey: ['cartData'], queryFn: getCartData }) as {
+    data: CartAPIDataType;
+    isSuccess: boolean;
+  };
+  const customData = isSuccess ? data.CUSTOM : [];
```

Comment:
```suggestion
  const { data: dataName, isSuccess } = useQuery({ queryKey: ['cartData'], queryFn: getCartData }) as {
    data: CartAPIDataType;
    isSuccess: boolean;
  };
  const customData = isSuccess ? data.CUSTOM : [];
```
여기도 이름을 wrap해서 사용하는게 더 좋을거같군요